packages/hyperion-autologging/src/ALInteractableDOMElement.ts

```diff
@@ -76,15 +77,34 @@ function elementHasEventHandler(node: HTMLElementWithHandlers, eventName: HTMLEl
 }
 
 function ignoreInteractiveElement(node: HTMLElement) {
-  const innerHeight: number = window.innerHeight;
-  const innerWidth: number = window.innerWidth;
   return (
     node.tagName === 'BODY' ||
-    node.tagName === 'HTML' ||
-    (node.clientHeight === innerHeight && node.clientWidth === innerWidth)
+    node.tagName === 'HTML'
   );
 }
 
+const isFunctionBodyNoop = (body: string): boolean => {
+  // Extract body between { } via regex and trim to determine noop
+  const match = body.match(/{(?<body>(.*))}/);
+  return match?.groups?.body?.trim() === '';
+}
+
+// Return a bool if the listener function body is a noop.
+function isListenerNoop(listener: EventListenerOrEventListenerObject): boolean | null {
+  if (typeof listener === 'function') {
```

Comment:
use `isEventListenerObject` function which is more accurate and consistent. Then use `getFunctionInterceptor(func)?.getOriginal() ?? func` as the target func.